src/puppetlabs/puppetdb/command.clj

```diff
@@ -450,9 +443,11 @@
   (init [this context]
     (let [response-chan (async/chan 1000)
           response-mult (async/mult response-chan)
-          response-chan-for-pub (async/chan)]
+          response-chan-for-pub (async/chan)
+          concurrent-writes (get-in (get-config) [:command-processing :concurrent-writes])]
```

Comment:
The opposite of this is true (noted in other comments). This is very important for performance with spinning disks. This code will be staying.